src/dart_pipeline/process.py

```diff
@@ -803,59 +1190,45 @@ def process_worldpop_pop_count_data(
     - EPSG:4326: https://epsg.io/4326
     - EPSG = European Petroleum Survey Group
     """
-    source = "sociodemographic/worldpop-count"
+    sub_pipeline = 'sociodemographic/worldpop-count'
     country = get_country_name(iso3)
-    print("Year:       ", year)
-    print("Country:    ", iso3)
-    print("Resolution: ", rt)
-    print("")
+    logging.info('year:%s', year)
+    logging.info('iso3:%s', iso3)
+    logging.info('resolution_type:%s', rt)
 
-    base_path = source_path(
-        source,
-        f'GIS/population/by-country/{iso3}/{country.replace(' ', '_')}_100m_Population',
-    )
-    filename = Path(f"{iso3}_{rt}_v2b_{year}_UNadj.tif")
-    print(f'Processing "{filename}"')
-    src = rasterio.open(base_path / filename)
-    # Get the affine transformation coefficients
-    transform = src.transform
+    filename = Path(f'{iso3}_{rt}_v2b_{year}_UNadj.tif')
+    path = source_path(sub_pipeline, iso3) / filename
+    logging.info('importing:%s', path)
+    src = rasterio.open(path)
     # Read data from band 1
     if src.count != 1:
-        raise ValueError(f"Unexpected number of bands: {src.count}")
+        raise ValueError(f'Unexpected number of bands: {src.count}')
     source_data = src.read(1)
 
     # Replace placeholder numbers with 0
     # (-3.4e+38 is the smallest single-precision floating-point number)
     df = pd.DataFrame(source_data)
     population_data = df[df != MIN_FLOAT]
-    """
-    Sanity check: calculate the total population
-    Google says that Vietnam's population was 96.65 million (2020)
+    population = population_data.sum().sum()
+    logging.info('population:%s', population)
 
-    VNM_pph_v2b_2020.tif
-    90,049,150 (2020)
-
-    VNM_pph_v2b_2020_UNadj.tif
-    96,355,010 (2020)
+    # Initialise an output data frame
+    df = pd.DataFrame(columns=OUTPUT_COLUMNS)
 
-    VNM_ppp_v2b_2020.tif
-    90,008,170 (2020)
+    # Populate output data frame
+    df.loc[0, 'iso3'] = iso3
+    df.loc[0, 'admin_level_0'] = country
+    df.loc[0, 'year'] = year
+    df.loc[0, 'metric'] = 'population'
+    df.loc[0, 'unit'] = 'people'
```

Comment:
unitless

Reply:
Is this only one row? I think for all outputs, only the standard [schema output](https://dart-pipeline.readthedocs.io/en/latest/schema.html) is required, with some exceptions (e.g. weekly aggregation) as discussed in the meeting